defrag/modules/helpers/broker.py

```diff
@@ -0,0 +1,76 @@
+from dataclasses import dataclass
+from defrag.modules.db.redis import RedisPool
+from pottery import RedisDeque
+from typing import Any, Coroutine, Dict, Generator, List, Optional
+import asyncio
+
+
+@dataclass
+class Message:
+    timestamp: float
+    sender: str
+    addressee: str
```

Comment:
Why do we have an addressee? My thought was we would implement some sort of "top to bottom" communication, where we send a message to a lot of people. This looks, combined with the tests, more like a messaging service, which I feel we have no need (and the required skills for making it secure and private) for.

Reply:
Don't worry, this is a "top to bottom" communication service. We simply need to know which entity sent the message and which entity it is meant for, that is all. "sender" and "addressee" are just simple words for my simple mind to remember easily.

Example:
- sender: "telegram:bot,user:jens"
- addressee: "telegram:subscribed_bots,matrix:subscribed_bots"
- text: "Today there will be an outage due to a meteor impact in the Nuremberg region. Expect 1-2 hours with oS infra down, starting at 12 pm."

Reply:
Live this message :joy: